features/step_definitions/page_steps.rb

```diff
@@ -256,13 +256,13 @@
 
 Then "there should be an indication of unpublished changes" do
   step %{I wait for ajax requests to complete}
-  within '.mercury-toolbar' do
+  within '.mercury-primary-toolbar' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.